src/data/frontend/profile/mappers.ts

```diff
@@ -52,6 +53,14 @@ export const mapSpecialization = (specialization: JobSpecialization) => {
   }
 }
 
+export const mapOptions = <T extends { name: string }>(
+  items: T[],
```

Comment:
**praise/thought:** Nice, however we have to be careful, as sometimes we will need for example `item.id` as value instead of name. Or `name` will be formatted, so it's nicely displayed to user but value will be in ugly format that we will use in database (like enums)